app/upgrades/drs-2/constants.go

```diff
@@ -0,0 +1,19 @@
+package drs2
+
+import (
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	"github.com/dymensionxyz/rollapp-evm/app/upgrades"
+	claimstypes "github.com/evmos/evmos/v12/x/claims/types"
+)
+
+const (
+	UpgradeName = "upgrade-drs-2"
```

Comment:
can we just call it `drs-2` ? 
I think it may require change in the rdk no?

Reply:
either way not critical for now

Reply:
done